ui/tests/integration/components/masked-input-test.js

```diff
@@ -96,4 +96,26 @@ module('Integration | Component | masked input', function (hooks) {
     const unMaskedValue = document.querySelector('.masked-value').value;
     assert.strictEqual(unMaskedValue, this.value);
   });
+
+  test('it renders correctly with empty string provided as value', async function (assert) {
+    await render(hbs`
+      <MaskedInput
+        @name={{"key"}}
+        @value={{""}}
```

Comment:
just a handlebars syntax note - you can pass in a string directly as an arg and don't need to wrap it in curly brackets `@someArg="hello"`